plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectWizard.java

```diff
@@ -129,12 +110,5 @@ protected void retrieveConfigurationValues() {
   @Override
   public void init(IWorkbench workbench, IStructuredSelection selection) {
     this.workbench = workbench;
-    if (config.getCloudSdkLocation() == null) {
-      File location = CloudSdkPrompter.getCloudSdkLocation(getShell());
-      // if the user doesn't provide the Cloud SDK then we'll error in performFinish() too
-      if (location != null) {
-        config.setCloudSdkLocation(location);
```

Comment:
Looks like you can get rid of AppEngineProjectConfig.cloudSdkLocation too

Reply:
Yeah, that has been removed.